contrib/scripts/unloaddb.sh

```diff
@@ -0,0 +1,387 @@
+#!/bin/bash
+#
+#
+#  Copyright 2016 CUBRID Corporation
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+verbose="no"   # set 'yes' for verbose mode
+max_num_proc=16
+num_proc=8
+table_size=()
+table_selected=()
+num_tables=0
+slot_selected=0
+# following two variable are depend on max_num_proc
+slot_size=(0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0)
+num_tables_slot=(0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0)
+database=""
+user="-u dba"
+pass=""
+total_pages=0
+filename=""
+target_dir=$(pwd)
+from_file=0
+num_args_remain=0
+
+slot=()
+
+function show_usage ()
+{
+         echo "Usage: $0 [OPTIONS] [database]"
+         echo " OPTIONS"
+         echo "  -t arg  Set number of parallel process; default 8"
```

Comment:
I like to add maximum value description. (e.g. max 16)

Reply:
Thank you for your comments, I missed to delete it.
Fixed.